fastlane/Fastfile

```diff
@@ -589,34 +560,30 @@ platform :ios do
     )
   end
 
-  #####################################################################################
-  # trigger_beta_build
-  # -----------------------------------------------------------------------------------
-  # This lane triggers a beta build on CI
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane trigger_beta_build [branch_to_build:<branch_name>]
+  # This lane triggers a beta build on CI.
   #
-  #####################################################################################
-  lane :trigger_beta_build do |options|
+  # @param [String] branch_to_build The git branch the CI build should run on.
+  #
+  # @example Running the lane
+  #          bundle exec fastlane trigger_beta_build branch_to_build:main
+  #
+  lane :trigger_beta_build do |branch_to_build|
```

Comment:
With this syntax, `branch_to_build` is still an unnamed parameter (as in: call site won't need a keyword preceding the parameter in the call). You probably meant:
```suggestion
  lane :trigger_beta_build do |branch_to_build:|
```

Reply:
Nice catch -- updated on 2ec32f7ec4ba37eceb2611bc6a20500c115e98b3